incomplete type error encountered

The following program `sample.go` triggers a panic:

```go
package main

import (
	"github.com/traefik/yaegi/interp"
	"github.com/traefik/yaegi/stdlib"
)

const codeStr = `
import "github.com/gdbu/scribe"
	
func main(){
	out := scribe.New("test")
	out.Success("Yes")
}
`

func main() {
	var opts interp.Options
	opts.GoPath = "/path/to/go"
	i := interp.New(opts)
	i.Use(stdlib.Symbols)

	var err error
	if _, err = i.Eval(codeStr); err != nil {
		panic(err)
	}

	if _, err = i.Eval(`main()`); err != nil {
		panic(err)
	}
}

```

Expected result:
```console
$ go run ./sample.go
// Yes
```

Got:
```console
$ go run ./sample.go
// panic: /Users/panda/go/src/github.com/hatchify/colors/attribute.go:5:8: incomplete type 
```

--- a/_test/const25.go
+++ b/_test/const25.go
@@ -1,0 +1,11 @@
+package main
+
+const (
+	FGBlack Attribute = iota + 30
+)
+
+type Attribute int
+
+func main() {
+	println(FGBlack)
+}

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -82,8 +82,15 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 					i--
 				}
 				dest := a.child[i]
-				if dest.typ != nil && !isInterface(dest.typ) {
-					// Interface type are not propagated, and will be resolved at post-order.
+				if dest.typ == nil {
+					break
+				}
+				if dest.typ.incomplete {
+					err = n.cfgErrorf("invalid type declaration")
+					return false
+				}
+				if !isInterface(dest.typ) {
+					// Interface types are not propagated, and will be resolved at post-order.
 					n.typ = dest.typ
 				}
 			case binaryExpr, unaryExpr, parenExpr:

